README.md

```diff
@@ -1,3 +1,42 @@
 # medic-builds-repo
 
 Design doc for builds repository.
+
```

Comment:
+1.  I'd like to take this moment to complain about https://github.com/medic/horticulturalist/pull/1/files#diff-1fdf421c05c1140f6d71444ea2b27638L69